dask/dataframe/io/tests/test_io.py

```diff
@@ -309,6 +309,23 @@ def test_from_pandas_convert_string_config():
     assert_eq(df_pyarrow, ddf)
 
 
+@pytest.mark.skipif(PANDAS_GT_200, reason="Requires pandas<2.0")
```

Comment:
There are some other tests that set `dataframe.convert_string` to `True` that we'll also want to apply this `skipif` logic too